### PR TITLE
fixed an issue with datetimeimmutable comparaison

### PR DESCRIPTION
**Pull request by Kharhamel**
This pr solves #179

### CODE REVIEW

File: lib/DateTimeImmutable.php [hunk 1]
```diff
@@ -28,14 +28,14 @@ class DateTimeImmutable extends \DateTimeImmutable
...
-        parent::__construct();
+        parent::__construct($time, $timezone);
         $this->innerDateTime = new parent($time, $timezone);
     }
 
     //switch from regular datetime to safe version
     private static function createFromRegular(\DateTimeImmutable $datetime): self
     {
-        $safeDatetime = new self();
+        $safeDatetime = new self($datetime->format('Y-m-d H:i:s'), $datetime->getTimezone()); //we need to also update the wrapper to not break the operators '<' and '>'
```
Comment (bendavies) [inline lib/DateTimeImmutable.php]: what about microseconds?

**Comment by kunicmarko20** [comment]
@Kharhamel thank you for fast response, can we potentially get a patch release with this?

File: lib/DateTimeImmutable.php (hunk 1 above)
Comment (Kharhamel) [reply lib/DateTimeImmutable.php]: damn i forgot about theses

Comment (kunicmarko20) [reply lib/DateTimeImmutable.php]: So milliseconds can't be used with < 7.4 because `::createFromFormat` doesn't accept it, microseconds should be used.

**Pull request merged**